testutils/ptest2/pom.xml

```diff
@@ -115,20 +115,24 @@ limitations under the License.
       <artifactId>velocity</artifactId>
       <version>1.7</version>
     </dependency>
+    <dependency>
+      <groupId>com.fasterxml.jackson</groupId>
+      <artifactId>jackson-bom</artifactId>
+      <version>${jackson.version}</version>
+      <type>pom</type>
+      <scope>import</scope>
+    </dependency>
```

Comment:
This needs to be moved to a `dependencyManagement` section above `dependencies`.